server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -347,8 +347,8 @@ public void run() {
         }
         if (manager.tserverSet.getCurrentServers().equals(currentTServers.keySet())) {
           Manager.log.debug(String.format("[%s] sleeping for %.2f seconds", store.name(),
-              Manager.TIME_TO_WAIT_BETWEEN_SCANS / 1000.));
-          eventListener.waitForEvents(Manager.TIME_TO_WAIT_BETWEEN_SCANS);
+              manager.getWaitTimeBetweenScans() / 1000.));
+          eventListener.waitForEvents(manager.getWaitTimeBetweenScans());
```

Comment:
I still think you need to use a local variable in here, so the log message isn't logging a different value than what's actually being used, which makes debugging a pain.

And, this, as well as the occurrence above are still in the same `while (manager.stillManager())` loop. It would be weird if the value changed from one operation to another in that same loop. They should share the same local value.

Reply:
Created a local variable at the top of the loop so that the value for the property is the same for the entire iteration, in ff6923e